recipes/flite/build.sh

```diff
@@ -0,0 +1,7 @@
+set -exou
+
+./configure --with-vox=cmu_us_kal16 --enable-shared --prefix="$(pwd)/build"
+
+make
+
```

Comment:
Is there a `make check` or `ctest`?

Reply:
No, there isn't. There is an unmerged PR from gentoo: https://github.com/festvox/flite/pull/90 . I don't feel like maintaining an unmerged PR.

But the test suite is actually there, so it can be run manually. But it can't be run with a simple `make something`. It has to make the test and the run it from CLI. But I don't think this is worth it. The test already present in this recipe tests that the main binary is working. I think that's enough, isn't it?

Reply:
> The test already present in this recipe tests that the main binary is working. I think that's enough, isn't it?

The more tests the better but this is probably good enough to merge and get the first package out. Just add an issue in the feedstock to try adding more tests in the future.